Makefile

```diff
@@ -50,9 +52,10 @@ stop_staging:
 	$(staging_compose_cmd) stop
 
 generate_maps_staging: build_migrate_staging
-	$(staging_compose_cmd) run --rm web python3 manage.py sync_geofiles
-	$(staging_compose_cmd) run --rm web python3 manage.py generate_meteorological_raster_cities
-	$(staging_compose_cmd) run --rm web python3 manage.py generate_mapfiles
+	$(staging_compose_cmd) run --rm staging_web python3 manage.py sync_geofiles
+	# $(staging_compose_cmd) run --rm staging_web python3 manage.py generate_meteorological_raster_cities
```

Comment:
as it is temporary, could you add a comment above with the number of the issue related to that?